offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -33,8 +21,7 @@ import java.util.concurrent.TimeUnit
  *
  * Implements {@link OfferExporter} and is responsible for converting an offer into PDF Format
  *
- * @since: 1.0.0
- * @author: Jennifer Bödker
+ * @since: 1.0.0 @author: Jennifer Bödker
```

Comment:
might not be the change we want ;)
```suggestion
 * @since 1.0.0 
 * @author Jennifer Bödker
```

Reply:
Yes, you are right. The `:` behind the tags confused the formatting.